testdata/dubious/sync-pool-pointers.go

```diff
@@ -0,0 +1,10 @@
+package pkg
+
+import "sync"
+
+func fn() {
+	pool := sync.Pool{}
```

Comment:
I'm not checking for `*sync.Pool` here, do you want this test duplicated to include that?